frontend/src/modules/news/components/BrowseNews/constant.ts

```diff
@@ -0,0 +1,107 @@
+export const constants = [
```

Comment:
this `constant` should come with `types`, please defined a type in `model` directory e.g.

```ts
export type BrowseNewsModel = {
id: string,
title: string,
...
}
```